torch/_dynamo/compiled_autograd.py

```diff
@@ -534,12 +534,15 @@ def enable(compiler_fn):
         torch._C._dynamo.compiled_autograd.set_verbose_logger(cpp_verbose_log_fn)
     global compiled_autograd_enabled
     compiled_autograd_enabled = True
+    prior_config = torch._dynamo.config.compiled_autograd
+    torch._dynamo.config.compiled_autograd = True
     try:
         with torch.autograd.set_multithreading_enabled(False):
             yield
     finally:
         if not prior:
             compiled_autograd_enabled = False
+        torch._dynamo.config.compiled_autograd = prior_config
         torch._C._dynamo.compiled_autograd.set_autograd_compiler(prior)
```

Comment:
maybe we can rebase on top of https://github.com/pytorch/pytorch/pull/138113 to avoid potential merge conflict?